src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -135,9 +135,17 @@ public void OnDataReceived(string jsonRpcPayload)
                         case CustomProtocolNames.CommandExecuted:
                             HandleCommandExecutedRequest(id, paramsJson);
                             break;
+
+                        case CustomProtocolNames.GetCapabilities:
```

Comment:
Happened to come across this. Why do we need Custom capabilities protocol when language server has a well defined initialize protocol that includes capabilities? I understand the need we need to get this done quickly but we should follow the protocol whenever we can

Reply:
https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#initialize

Reply:
Chatted offline. Given the need of the hour, it should be okay to get this in as long as we follow up to do it the way protocol specifies.

Reply:
Tracking with #1843